Clean installation of preact-cli fails to build

I was working on upgrading to the latest version last night to debug a separate problem and decided to go ahead and update from 1.3.0 to 1.4.0, as well as a totally new install on a fresh install of node. 

node -v: v6.9.5
npm -v: 5.1.0
preact -v: preact-cli 1.4.0 (upgraded from 1.3.0)

---

node -v: v8.2.1
npm -v: 5.3.0
preact -v: preact-cli 1.4.0 (new install)

I generated a new project using `create`. Build fails with error:

```
Build failed!

(webpack)-dev-server/client?http://0.0.0.0:8080/
Module not found: Error: Can't resolve 'loglevel' in '/Users/me/dev/preact-cli-test/node_modules/webpack-dev-server/client'
 @ (webpack)-dev-server/client?http://0.0.0.0:8080/ 4:10-29
 @ multi ./~/preact-cli/lib/lib/entry webpack-dev-server/client?http://0.0.0.0:8080/ webpack/hot/dev-server?http://0.0.0.0:8080/
```

Seems to be a bug in webpack-dev-server (https://github.com/webpack/webpack-dev-server/issues/984). Supposedly, falling back to old versions would work but one user reported still seeing the error while using `2.4.2` (so slightly older than this project's dep `2.4.5`). That would suggest that that bug is also affecting this project. If that issue is as pervasive as it seems, this may affect every new installation.

### Workaround
Fortunately the workaround is to simply install `loglevel` on your own project, until preact-cli upgrades. 

`npm i -D loglevel` or `yarn add -D loglevel`

### Solution
In the long term, preact-cli probably needs to upgrade to webpack-dev-server +`2.6.0` but I'll leave that to the team to further analyze and see if its compatible.

--- a/package.json
+++ b/package.json
@@ -156,7 +156,7 @@
     "url-loader": "^0.5.8",
     "webpack": "^2.3.3",
     "webpack-chunk-hash": "^0.4.0",
-    "webpack-dev-server": "^2.4.5",
+    "webpack-dev-server": "^2.6.1",
     "webpack-plugin-replace": "^1.1.1",
     "which": "^1.2.14",
     "yargs": "^8.0.1"

